src/main/java/com/rackspace/salus/monitor_management/services/MonitorManagement.java

```diff
@@ -2311,13 +2311,18 @@ public RenderedMonitorTemplate renderMonitorTemplate(UUID monitorId, String reso
         final String renderedContent = getRenderedContent(monitor.getContent(), resource);
         renderedMonitorTemplate.setRenderedContent(renderedContent);
       } catch (InvalidTemplateException e) {
-        log.warn("Unable to render content='{}' of monitor={} for resource={}",
+        log.error("Unable to render content='{}' of monitor={} for resource={}",
             monitor.getContent(), monitor, resource, e);
-        invalidTemplateErrors.increment();
+        throw new IllegalArgumentException(
+            String.format("Unable to render content=%s of monitor=%s for resource=%s",
+                monitor.getContent(), monitor, resource));
```

Comment:
The whole monitor in the exception message will be too much, so I'd suggest just:
```suggestion
            String.format("Unable to render content=%s for resource=%s",
                monitor.getContent(), resource));
```